src/main/scala/beam/agentsim/agents/MobilityRequest.scala

```diff
@@ -35,6 +40,39 @@ case class MobilityRequest(
       case Some(p) => p.personId.toString
       case None    => "None"
     }
-    s"${formatTime(time)}|$tag|${personid}|${activity.getType}| => ${formatTime(serviceTime)}"
+//    s"${formatTime(baselineNonPooledTime)}|$tag|${personid}|${activity.getType}| => ${formatTime(serviceTime)}"
+    s"${baselineNonPooledTime}|$tag|${personid}|${activity.getCoord}| => ${serviceTime}"
+  }
+  override def equals(that: Any): Boolean = {
```

Comment:
This is tricky and can cause different issues in the future when you have a collection (List/Vector/Map) of `MobilityRequest` and want to find an instance of `MobilityRequest`.

Reply:
Agree, but there's an important reason for doing it.